<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:#f1c40f}</string>
   </property>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #f1c40f;
background-color: #2c3e50;</string>
    </property>
    <property name="text">
     <string>n11 Scrapper</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="categoryNameField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;</string>
    </property>
    <property name="text">
     <string>Enter category:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="enterPageCountField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter page count:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startScrappingButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>295</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:20px;
background-color: #c0392b;
color: #ecf0f1;
}
QPushButton::hover
{
background-color: #ecf0f1;
color: #c0392b;

}</string>
    </property>
    <property name="text">
     <string>Scrap</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="categoryInputField">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border:2px solid #2c3e50;
background:#ecf0f1;
color:#000000;
}

</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>bilgisayar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pageCountInputField">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border:2px solid #2c3e50;
background:#ecf0f1;
color:#000000;
}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>less than 51</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
